### ISSUE TITLE
tutorial 16 - merge-env! with nonexistent dirs

### ISSUE THREAD

**Issue by gonewest818** [magomimmo/modern-cljs#429]
In tutorial 16 there’s a note that says it's fine to do a `(merge-env! :source-paths dirs)` where `dirs` contains nonexistent paths, because there should be no error.  But that doesn't seem to be the case for me. 

In the following example I'm just launching a repl and merging in a nonexistent directory named "d":

```
$ boot repl
nREPL server started on port 61064 on host 127.0.0.1 - nrepl://127.0.0.1:61064
REPL-y 0.3.7, nREPL 0.2.12
Clojure 1.8.0
Java HotSpot(TM) 64-Bit Server VM 1.8.0_112-b16
...
boot.user=> (merge-env! :source-paths #{"d"})

java.nio.file.NoSuchFileException: d
```

This is boot 2.7.1

In the tutorial this means I can't `boot tdd` with no arguments after adding the "-t" option because the default value for "-t" includes nonexistent dirs `test/clj` and `test/cljs`.  I know it's trivial to just pass "-t test/cljc" and so of course that's how I'm proceeding.  But the behavior I'm seeing directly contradicts what the tutorial suggests ought to happen.

**Comment by GiovanniFerrari** [comment]
You are right. That note is true only until boot version 2.5.5. I will correct the tutorial.

**Comment by gonewest818** [comment]
Looks like you corrected this in commit 49c19426ba5de0e9eb3fc584c5afe6a2f79b6b72 so I'm going to mark this closed.  Thanks!
